src/app_service/service/accounts/service.nim

```diff
@@ -334,6 +334,12 @@ proc login*(self: Service, account: AccountDto, password: string): string =
         "Enabled": true,
         "OpenseaAPIKey": OPENSEA_API_KEY_RESOLVED
       },
+      "TorrentConfig": {
+        "Enabled": false,
+        "DataDir": joinPath(main_constants.defaultDataDir(), "data", "archivedata"),
+        "TorrentDir": joinPath(main_constants.defaultDataDir(), "data", "torrents"),
+        "Port": 9025
+      }
```

Comment:
Can this one refer to the network_constants, or they can have different values?

Reply:
Can definitely move it to constants. This however provides a way to change it on a per desktop client instance basis.